charts/umbrella/Chart.yaml

```
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 2.14.2
+version: 2.15.0
```

this change should certainly be reflected by a major
```suggestion
version: 3.0.0
```